rs/rust_canisters/xnet_test/Cargo.toml

```diff
@@ -13,8 +13,8 @@ path = "src/main.rs"
 [dependencies]
 candid = { workspace = true }
 futures = { workspace = true }
-ic-cdk = { workspace = true }
-ic-cdk-macros = { workspace = true }
+ic-cdk = { git = "https://github.com/dfinity/cdk-rs", branch = "next" }
+ic-cdk-macros = { git = "https://github.com/dfinity/cdk-rs", branch = "next" }
```

Comment:
Do we need to pin a specific version here to avoid things breaking if the API changes?

Reply:
(potentially with a comment / ticket to unpin once the API is stable)

Reply:
Yeah, I've had to pin it to the same commit as bazel and the other crate, because the latest commit on the branch has a `&Call` field in `CallFutureState` (the `Future` returned by `call()`), so that the borrow checker won't let you collect multiple futures and join them all. Or maybe I'm just too tired to figure out how to do it.

Reply:
Makes sense. Thanks!